tool/dart_model_generator/lib/definitions.dart

```diff
@@ -237,21 +237,59 @@ static Protocol handshakeProtocol = Protocol(
                   type: 'bool', description: 'Whether the entity is static.'),
             ]),
         Definition.clazz('QualifiedName',
-            description: 'A URI combined with a name.',
+            description: 'A URI combined with a name and scope referring to a '
+                'declaration. The name and scope are looked up in the export '
+                'scope of the URI.',
             createInBuffer: true,
             properties: [
               Property('uri',
                   type: 'String',
                   description: 'The URI of the file containing the name.'),
-              Property('name', type: 'String', description: 'The name.'),
+              Property('scope',
+                  type: 'String',
+                  description: 'The optional scope to look up the name in.',
+                  nullable: true),
+              Property('name',
+                  type: 'String',
+                  description: 'The name of the declaration to look up.'),
+              Property('isStatic',
```

Comment:
I guess this will probably end up as an enum, since we might want to distinguish more types of thing (e.g. getter vs setter vs field), and more might arrive later. Maybe a TODO since it's easy to change for now?

Reply:
I don't think it ever matters whether it is a getter/setter/field for the purposes of this object, it is just telling you how to look up a declaration unambiguously, which does require knowing if it should be looked up in the static or instance member scope.

Reply:
Doesn't matter for right now, but I was thinking of a case where you have e.g. a subclass with a setter and superclass with a getter; you need to say setter or getter to disambiguate between the two. It's a bit of an awkward corner :) anyway, we can figure that out later if needed. Thanks.